ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/platform.j2

```diff
@@ -1,13 +1,41 @@
-{% if platform is defined and platform is not none %}
+{% if platform is arista.avd.defined %}
 
 # Platform
+{%     if platform.trident is arista.avd.defined or platform.sand is arista.avd.defined %}
 
 ## Platform Summary
-{%     if platform.trident is defined and platform.trident is not none %}
-{%       if platform.trident.forwarding_table_partition is defined and platform.trident.forwarding_table_partition is not none %}
+{%         if platform.trident is arista.avd.defined %}
 
-   The platform trident forwarding-table partition is {{ platform.trident.forwarding_table_partition }}.
-{%       endif %}
+### Platform Trident Summary
+
+| Settings | Value |
+| -------- | ----- |
+{%             for tridentsetting in platform.trident %}
```

Comment:
Looks Good to me! Just a minor suggestion, you can use ```natural_sort``` filter on every for loop to make the document look better.

Reply:
Sure, let me add it and regenerate !